src/software/ai/hl/stp/tactic/shoot_goal_tactic_test.cpp

```diff
@@ -51,7 +51,7 @@ TEST(ShootGoalTacticTest, robot_will_commit_to_a_shot_until_it_is_entirely_block
 
     ShootGoalTactic tactic =
         ShootGoalTactic(world.field(), world.friendlyTeam(), world.enemyTeam(),
-                        world.ball(), Angle::fromDegrees(6), std::nullopt, false);
+                        world.ball(), Angle::fromDegrees(4), std::nullopt, false);
```

Comment:
I ought to check by hand, but I ended up getting an open angle of 5.2 degrees for this unit test, which failed because the min_open_angle was too large.

Reply:
Would be good to check that out before we change this value just in case it exposes a bug or something.

Reply:
![image](https://user-images.githubusercontent.com/25271038/71539146-008fb680-28ed-11ea-9f57-1918e90807eb.png)
Looks correct to me

Reply:
I'm surprised that the angle is only 5.1 degrees. Visually it looks much larger but that might just be the scale

Reply:
Ok probably fine then, may have been an old bug.